types.go

```diff
@@ -51,7 +51,7 @@ type Runner interface {
 // RawMetricHandler is an interface that accepts a Metric for processing.  Raw refers to pre-aggregation, not
 // pre-consolidation.
 type RawMetricHandler interface {
-	DispatchMetric(ctx context.Context, m *Metric)
+	DispatchMetrics(ctx context.Context, m []*Metric)
```

Comment:
Might be better for GC if this was a slice of metrics, not of pointers. Maybe not, needs testing.

Reply:
It possibly would, but with the whole distributed thing, this should hopefully be less critical.  Once everything is stabilized, I may make another optimization sweep.